deploy/parameter/helm-values.yaml

```diff
@@ -20,7 +20,7 @@ requestTracing:
 
 workloadIdentity:
   enabled: true
-  disableGlobalServiceAccount: false
+  enableGlobalServiceAccount: false
```

Comment:
This is a verb. As a property name, it should be `globalServiceAccountEnabled`. Alternatively, you can have `useGlobalServiceAccount` or `globalServiceAccountAllowed`. Any preference?

Reply:
globalServiceAccountEnable is good to me